packages/create-cloudflare/src/workers.ts

```diff
@@ -145,10 +149,10 @@ async function updateFiles(ctx: Context) {
 	// update files
 	contents.packagejson.name = ctx.project.name;
 	contents.wranglertoml = contents.wranglertoml
-		.replace(/^name = .+$/m, `name = "${ctx.project.name}"`)
+		.replace(/^name\s*=\s*"<TBD>"$/m, `name = "${ctx.project.name}"`)
 		.replace(
-			/^compatibility_date = .+$/m,
-			`compatibility_date = "${new Date().toISOString().substring(0, 10)}"`
+			/^compatibility_date\s*=\s*"<TBD>"$/m,
```

Comment:
If a template add a comment after either the date or name these regexs won't match

Since you're matching the placeholder value and quotes, you can simply replace that and leave the rest of the line intact

Reply:
Agh! Of course. Good call.